### DISCUSSION TITLE
99.9% at #44 Stripes but Diff shows an black screen (perfect diff)

### DISCUSSION THREAD

**Discussion by rubenbelem** [cssbattle/forum#343] [Bugs reports]
Hey, I made a submission to #44, having a black screen at diff like this:
![image](https://user-images.githubusercontent.com/8121991/222877883-a334e9da-83a3-4dff-82eb-88e7d537d3f2.png)

But got 99.9%. Seems like a bug.

Here is the solution I attempted:

`<div w><div></div><div></div><div></div><div></div><div></div></div><div c l></div><div c r></div><style>body,div[c]{background:#1a4341}div[c],div[w]{position:absolute}body{margin:0}div[w]{top:40px;left:120px;width:160px;height:200px;flex-direction:columm;align-items:strech}div[w]>div{background:#f3ac3c;height:20px;margin-top:20px}div[c]{width:300px;height:300px;border-radius:50%}div[l]{right:-150px}div[r]{left:-150px}</style>`

**Comment by Gabb192** [comment]
Sometimes changing color helps to find 100%. In this case switching from `#1A4341` to `#1B4441` worked.

`<div w><div></div><div></div><div></div><div></div><div></div></div><div c l></div><div c r></div><style>body,div[c]{background:#1b4441}div[c],div[w]{position:absolute}body{margin:0}div[w]{top:40px;left:120px;width:160px;height:200px;flex-direction:columm;align-items:strech}div[w]>div{background:#f3ac3c;height:20px;margin-top:20px}div[c]{width:300px;height:300px;border-radius:50%}div[l]{right:-150px}div[r]{left:-150px}</style>`.

A very basic optimization of the code could be this:

`<div w><div></div><div></div><div></div><div></div><div></div></div><div c l></div><div c r><style>*{background:#1b4441;margin:0}[c],[w]{position:absolute;width:300;height:300;border-radius:50%}[w]{top:40;left:120;width:160;height:200}[w]>*{background:#f3ac3c;height:20;margin:20 0}[l]{right:-150}[r]{left:-150`

@rubenbelem

**Reply by WildAboutWeb** [reply]
Can this be updated in the exercise? 
#1A4341 --> #1B4441
